packages/core/src/components/TextField/Mask.example.jsx

```diff
@@ -0,0 +1,50 @@
+import TextField, { unmaskValue } from './TextField';
+import React from 'react';
+import ReactDOM from 'react-dom';
+// import { unmaskValue } from '@cmsgov/design-system-core';
+
+function handleBlur(evt, mask) {
+  console.log('Unmasked value:', unmaskValue(evt.target.value, mask));
```

Comment:
`console.log` left in

Reply:
Actually, this in intentional, right? Nevermind.

Reply:
But is the comment above `handleBlur` intentional for the example?

Reply:
Yeah, these are all hacks to demonstrate how someone might implement this 😅 Open to better ideas!